Pilot/PilotLoggerTools.py

```diff
@@ -11,79 +11,145 @@
 
 __RCSID__ = "$Id$"
 
-def createPilotLoggerConfigFile( filename = 'PilotLogger.cfg',
-                                 host = '',
-                                 port = '',
-                                 queuePath = '',
-                                 key_file  = '',
-                                 cert_file = '',
-                                 ca_certs = '',
-                                 fileWithID = ''):
-  """Helper function that creates proper configuration file.
-     The format is json encoded file with the following options included
+
+def createPilotLoggerConfigFile(filename='PilotLogger.json',
+                                loggingType='',
+                                localOutputFile='',
+                                host='',
+                                port='',
+                                url='',
+                                key_file='',
+                                cert_file='',
+                                ca_certs='',
+                                fileWithID='',
+                                queue=None,
+                                setup = 'Dirac-Certification'):
+  """Helper function that creates a proper configuration file.
+     The format is json encoded file with the following options included.
+     Arguments:
+      queue(dict): e.g. {"lhcb.test.*":{"Persitent":"False", "Ackonwledgement":"False"}}
```

Comment:
There are many more arguments.

Reply:
this funcition is mainly used for tests. I changed the descrpition for more explicit.